provider/execution/execution.go

```diff
@@ -299,8 +286,8 @@ func (e TableExecutor) resolveResources(ctx context.Context, meta schema.ClientM
 		diags     diag.Diagnostics
 	)
 
-	for _, o := range objects {
-		resource := schema.NewResourceData(e.Db.Dialect(), e.Table, parent, o, e.metadata, e.executionStart)
+	for i := range objects {
```

Comment:
This change seems unnecessary, no?

Reply:
If the `interface{}` isn't a pointer then this change is necessary (and more future proof) due to the way `for` works in go.